core/tests/ts-integration/tests/api/contract-verification.test.ts

```diff
@@ -12,7 +12,7 @@ const DATE_REGEX = /\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}(\.\d{6})?/;
 
 const ZKSOLC_VERSION = 'v1.5.10';
 const SOLC_VERSION = '0.8.26';
-const ZK_VM_SOLC_VERSION = 'zkVM-0.8.26-1.0.1';
+const ZK_VM_SOLC_VERSION = 'zkVM-0.8.26-1.0.2';
```

Comment:
is this related?

Reply:
Nope, but it was a fix in main related to it

Reply:
Without it, for sometimes CI was completely broken